prow/monitoring/BUILD.bazel

```diff
@@ -18,41 +17,55 @@ release(
 load("@io_bazel_rules_k8s//k8s:object.bzl", "k8s_object")
 
 k8s_object(
-   name = "alertmanager_crd",
-   cluster = "{STABLE_PROW_CLUSTER}",
-   kind = None,
-   template = "@operator_framework_community_operators//:upstream-community-operators/prometheus/alertmanager.crd.yaml",
+    name = "alertmanager_crd",
```

Comment:
Doing a update-bazel.sh with the new bazel version dragged these changes in, I see them on @xmudrii's PR (https://github.com/kubernetes/test-infra/pull/11832#discussion_r292477000) as well. Any idea what's going on?

Reply:
I think this: https://github.com/kubernetes/test-infra/pull/12969#discussion_r292975352